live/app/templates/components/pix-logo.hbs

```diff
@@ -1,4 +1,4 @@
 {{#link-to "index" class="pix-logo__link" title="Lien vers la page d'accueil de PIX"}}
-  <img class="pix-logo__image" src="{{rootURL}}images/pix-logo.svg" alt="Logo officiel de PIX (version bêta)">
```

Comment:
Après pas mal de recherche, le résumé :

- `{{rootURL}}` est essentiellement un concept d'`ember-cli`. Ce n'est pas trop supposé leaker dans le framework lui-même. C'est pour ça que ça n'est pas disponible sur les composants.
- Quand `ember-cli`voit des URLs d'assets, il va automatiquement les renseigner sous la forme qui va bien. Ça inclut notamment le fingerprinting, ou préfixer l'URL d'un CDN, ou des choses comme ça. C'est donc important que les URLs des assets soient identifiables par ember-cli.
- **La bonne pratique recommandée dans la [doc de ember-cli](https://ember-cli.com/asset-compilation) est d'écrire `src="images/pix-logo.svg"`**. Sans `/`, sans `{{rootURL}}`. Ember-cli se démerde derrière.

En revanche, si on crée un helper `src=(asset-url-for "images/pix-logo.svg")`, ember-cli ne va plus reconnaitre ces URLs comme des assets, et donc ça casse le fingerprinting, le préfixage d'un domaine, etc. Y'a des solutions pour contourner ça (par exemple quand on a besoin d'assets dont le nom est dynamique), ça implique un p'tit service pour charger la map des assets fingerprintés, et résoudre le nom en conséquence – mais le projet n'a clairement pas besoin de ça à ce stade.